docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +991,169 @@ eclagent:
       </sect3>
     </sect2>
   </sect1>
+
+  <sect1 id="LDAPSecurity_forConatiners">
+    <title>LDAP Security for Containers</title>
+
+    <para>Containerized LDAP support for HPCC Systems is accomplished by
+    externalizing the LDAP Active Directory Security Manager administrators
+    account credentials using Kubernetes or Hashicorp Vault secrets. As a
+    prerequisite to use these externalized credentials, you should be familiar
+    with setting up Kubernetes and/or Hashicorp Vault secrets.</para>
+
+    <para>The LDAP Administrator account is the only HPCC user account that
+    must have Active Directory administrator rights. This account should exist
+    in the configured "systemBasedn" branch of the Active Directory, typically
+    set to cn=Users.</para>
+
+    <sect2 id="Configuring_For_KubernetesSecrets">
+      <title>Configuring LDAP in Kubernetes</title>
+
+      <para>To create a secret in Kubernetes for the HPCC Administrator
+      account from a command line intereface to the Kubernetes, execute a
```

Comment:
"to the Kubernetes". Not sure if you are referring to the Kubernetes cluster, but "to the Kubernetes" should be changed.